desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -134,16 +134,26 @@ def query_result(request, notebook):
   history_uuid = query_execute['history_uuid']
   status = _check_status(request, operation_id=history_uuid)
   if status['query_status']['status'] == 'available':
-    response = _fetch_result_data(request, operation_id=history_uuid)
-    return response['result']
+    res = _fetch_result_data(request, operation_id=history_uuid, rows=max_rows)
+
+    # Truncate result and add first 'max_cols' number of columns of meta and data in table if greater present
```

Comment:
Better to not do any magic? Also looking at PrettyTable docs, the cols truncation could be done there.

Also: it there a way to just disable the line wrap in a slack message instead?

Reply:
Looks like there is no way to disable the line wrapping even in the slack message